### ISSUE TITLE
texlive depends on libX11

### ISSUE THREAD

**Issue by dkwo** [void-linux/void-packages#36737]
```
$ xbps-query -R texlive 
architecture: x86_64-musl
build-options: luajit 
filename-sha256: 3aff158e5a80daa19f1a0300ee9e2b75913754abda2b9310f1a1c94bd70bc106
filename-size: 6391KB
homepage: https://tug.org/texlive/
installed_size: 20MB
license: GPL-2.0-or-later
maintainer: fosslinux <fosslinux@aussies.space>
pkgname: texlive
pkgver: texlive-20210325_2
provides:
	tex-2021_1
repository: https://alpha.de.repo.voidlinux.org/current/musl
run_depends:
	dialog>=0
	ghostscript>=0
	perl-Tk>=0
	texlive-core>=0
	xbps-triggers>=0
	musl>=1.1.24_7
	libstdc++>=4.4.0_1
	libgcc>=4.4.0_1
	zlib>=1.2.3_1
	libpng>=1.6.2_1
	mpfr>=4.0.0_1
	cairo>=1.8.6_1
	pixman>=0.14.0_1
	libsynctex-20210325_2
	icu-libs>=70.1_1
	libX11>=1.2_1
```
Is this intentional @fosslinux ? or perhaps we can get rid of it for the next update cycle?

**Comment by fosslinux** [comment]
This is not an explicit dependency, I think it is pulled in by `libXaw-devel` (https://github.com/void-linux/void-packages/blob/master/srcpkgs/texlive/template#L73), which is needed for xdvi. We should probably split it out into a subpackage to avoid pulling X.
